atom/lib/AskForSelectionView.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const SelectListView = require('atom-space-pen-views').SelectListView;
+
+const Deferred = require('./Deferred');
+
+// TODO should this use a context menu instead of an overlay at the top of the
+// window?
+class AskForSelectionView extends SelectListView {
+  /**
+   * @param {Function} callback
+   */
+  initialize() {
+    super.initialize();
+    this.addClass('overlay from-top');
+    this.storeFocusedElement();
+    this.panel = atom.workspace.addModalPanel({ item: this });
+    this.deferred = new Deferred();
+    this.show();
+  }
+
+  viewForItem(item) {
+    return `<li>${item}</li>`;
+  }
+
+  confirmed(item) {
+    // The ImportJS CLI tool expects an index from the alternatives list, so we
+    // need to find the index.
+    // TODO: should we change the API so the CLI tool takes a path instead of an
+    // index?
```

Comment:
Yeah, I thought about that when rewriting the importer spec as well. I think it's fine with indexes for now (it also makes the command easier to construct, no need for escaping).